src/utils/get-ts-config.js

```diff
@@ -51,6 +52,26 @@ const getTsConfig = async (memo, fs, sourcePath, projectRoot) => {
     }
 };
 
+/**
+ * @param {string} jsoncString
+ * @param {string?} fileName
+ * @returns {any}
+ */
+function parseJsoncOrThrow(jsoncString, fileName){
+    /**
+     * @type {JSONC.ParseError[]}
+    */
+    let errors = [];
+    let parsingResult = JSONC.parse(jsoncString, errors, {allowTrailingComma: true});
```

Comment:
It seems as if the formatting is not enforced in this repo 🤔 I might find a minute today to add prettier to enforce code formatting

If I won't I will merge this as is though I think `.editorconfig` file is present and you can format the file according to it yourself using an IDE extension.

Reply:
Failed to make editorconfig format anything at all; I guess you're referring to max line length rule? Fixed that manually.